abci/versions.go

```diff
@@ -45,6 +45,7 @@ func (v Versions) GetForHeight(height int64) (Version, error) {
 	for _, version := range v {
 		if version.UntilHeight >= height {
 			selectedVersion = version
+			break
```

Comment:
@julienrbrt I think this is the desired behaviour, let me know if I'm wrong.

Reply:
we need to take the first matching one yes. However, we probably need to sort the versions per until height before, or fail is versions aren't sorted by until height.